index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="css/basic.css" />
+    <link rel="stylesheet" href="css/header.css" />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/footer.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <a href="index.html">
+        <img
+          src="img/panda_logo.png"
+          alt="판다마켓_로고_이미지"
+          class="pc_logo"
+        />
+        <img
+          src="img/panda_logo_text.png"
+          alt="판다마켓_로고_텍스트_이미지"
+          class="mo_logo"
+        />
+      </a>
+      <a href="login.html" class="login_button">로그인</a>
+    </header>
+
+    <main>
+      <section>
+        <div class="hero">
+          <div class="hero_title_box">
+            <h1 class="hero_title">일상의 모든 물건을<br />거래해 보세요</h1>
+            <a href="items.html" class="shop_button">구경하러 가기</a>
+          </div>
+        </div>
+      </section>
+
+      <article>
+        <div class="item_box">
+          <img src="img/Img_home_01.png" alt="인기상품_이미지" />
+          <div class="item_text_box">
+            <span>Hot item</span>
+            <h2>인기 상품을<br />확인해 보세요</h2>
+            <p>가장 HOT한 중고거래 물품을 판다 마켓에서 확인해 보세요</p>
+          </div>
+        </div>
+      </article>
+
+      <article>
+        <div class="item_box reverse">
+          <img src="img/Img_home_02.png" alt="구매상품_이미지" />
+          <div class="item_text_box text_align_right padding_right_60">
+            <span>Search</span>
+            <h2>구매를 원하는<br />상품을 검색하세요</h2>
+            <p>구매하고 싶은 물품은 검색해서 쉽게 찾아보세요</p>
+          </div>
+        </div>
+      </article>
+
+      <article>
+        <div class="item_box">
+          <img src="img/Img_home_03.png" alt="판매상품_이미지" />
+          <div class="item_text_box">
+            <span>Register</span>
+            <h2>판매를 원하는<br />상품을 등록하세요</h2>
+            <p>어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+          </div>
+        </div>
+      </article>
+
+      <aside>
+        <div class="banner">
+          <h1>믿을 수 있는<br />판다마켓 중고 거래</h1>
+        </div>
+      </aside>
+    </main>
+
+    <footer>
+      <div class="footer_container">
+        <div class="footer_copy"><p>ⓒcodeit - 2024</p></div>
+        <div class="footer_text">
+          <p><a href="privacy.html">Privacy Policy</a></p>
+          <p><a href="faq.html">FAQ</a></p>
+        </div>
+        <div class="footer_sns">
+          <a href="https://www.facebook.com" target="_blank"
+            ><img src="img/ic_facebook.png" alt="페이스북_아이콘_이미지"
+          /></a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer_sns">
          <li>
            <a href="https://www.facebook.com" target="_blank">
              <img src="img/ic_facebook.png" alt="페이스북_아이콘_이미지"/>
            </a>
          </li>
```